### PR TITLE
Trendmicro apex test

### PR DESCRIPTION
**Pull request by guykeller**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/39947

## Description
Fixed the bug by accommodating to new Python3 tag.
Confirmed by the customer.

### CODE REVIEW

File: Packs/TrendMicroApex/Integrations/TrendMicroApex/TrendMicroApex.py [hunk 1]
```diff
@@ -1,6 +1,4 @@
-import demistomock as demisto
```
Comment (DeanArbel) [inline Packs/TrendMicroApex/Integrations/TrendMicroApex/TrendMicroApex.py:1]: Why remove this import?

**Review by DeanArbel — APPROVED** [review]

File: Packs/TrendMicroApex/Integrations/TrendMicroApex/TrendMicroApex.py (hunk 1 above)
Comment (guykeller) [reply Packs/TrendMicroApex/Integrations/TrendMicroApex/TrendMicroApex.py]: fixed

**Review by yuvalbenshalom — APPROVED** [review]

**Comment by guykeller** [comment]
@yuvalbenshalom Same here - force merge please (no instance)

File: Packs/TrendMicroApex/ReleaseNotes/2_0_2.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### Trend Micro Apex One
+- Fixed an issue where getting a new JWT token would fail.
```
Comment (bakatzir) [inline Packs/TrendMicroApex/ReleaseNotes/2_0_2.md:4]:
```suggestion
- Fixed an issue where authenticating would fail intermittently.
```

**Pull request merged**